app/assets/stylesheets/common.scss

```diff
@@ -1036,4 +1036,12 @@ div.secondary-actions {
   margin-bottom: -1px;
 }
 
+/* Rules for traces */
+
+@include color-mode(dark) {
+  img.trace_image {
```

Comment:
Should this maybe be a custom utility class? The utility class name would tell more about what it does. Something like 'dark-invert-image'.

Reply:
this name would be wrong is non-dark style.
Or do I miss something?

Reply:
My point is not the suggested name but changing the name from a specific element name to a utility class name.